plex_trakt_sync/commands/inspect.py

```diff
@@ -37,8 +37,10 @@ def inspect(input):
     for guid in m.guids:
         print(f"  Guid: {guid}, Id: {guid.id}, Provider: {guid.provider}")
 
+    print(f"Payload: {m.to_json()}")
+
     try:
-        tm = trakt.find_by_media(m.guid)
```

Comment:
make this fix atomic commit. or even better separate pull request.

- https://www.freshconsulting.com/insights/blog/atomic-commits/